runbotics/runbotics-orchestrator-ui/src/translations/en/process.json

```diff
@@ -167,6 +167,7 @@
     "Process.Details.Modeler.Actions.Browser.Launch.Label": "Launch browser",
     "Process.Details.Modeler.Actions.Browser.Launch.Input": "Input",
     "Process.Details.Modeler.Actions.Browser.Launch.Headless": "Headless",
+    "Process.Details.Modeler.Actions.Browser.Open.Site.Input": "Open site",
```

Comment:
All Launch browser keys have this syntax `Process.Details.Modeler.Actions.Browser.Launch...`, I think `Process.Details.Modeler.Actions.Browser.Launch.Target` would fit better